src/lib/theme/theme.service.ts

```diff
@@ -62,6 +61,17 @@ export class VantageThemeService {
 
     // apply theme on storage or media query change
     merge(storageObserver, mediaObserver).subscribe((theme: VantageTheme) => this.applyTheme(theme));
+
+    // account for cached navigation
+    // needed for Firefox BFCache
+    window.addEventListener('pageshow', (pageTransition: PageTransitionEvent) => {
+      const localStorageDiffersActiveTheme: boolean =
+        this.localStorageTheme() && this.localStorageTheme() !== this._activeTheme;
```

Comment:
Let's save localStorageTheme to a local var since it's accessed 3 times.

Reply:
Sure